src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -337,7 +380,11 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
         .catch((err) => {
           this.isSaving = false;
           this.id = originalId;
-          if (err && err.message === OVERWRITE_REJECTED) return;
+          if (err && (
+            err.message === OVERWRITE_REJECTED ||
+            err.message === SAVE_DUPLICATE_REJECTED)) {
+            return;
+          }
```

Comment:
ehhh, I don't love suggestion, I'd look at that and be like, why not just combine them into an ||?

how about this new variation as a compromise?